js/basemap_selector.ts

```diff
@@ -0,0 +1,104 @@
+import type { RenderProps } from "@anywidget/types";
+import { css, html, PropertyValues, TemplateResult } from "lit";
+import { property, query } from "lit/decorators.js";
+
+import { legacyStyles } from "./ipywidgets_styles";
+import { materialStyles } from "./styles";
+import { loadFonts } from "./utils";
+import { LitWidget } from "./lit_widget";
+
+export interface BasemapSelectorModel {
+    basemaps: string[];
+    value: string;
+}
+
+export class BasemapSelector extends LitWidget<
+    BasemapSelectorModel,
+    BasemapSelector
+> {
+    static get componentName() {
+        return `basemap-selector`;
+    }
+
+    static styles = [
+        legacyStyles,
+        materialStyles,
+        css`
+            .row-container {
+                align-items: center;
+                display: flex;
+                height: 32px;
+                width: 200px;
+            }
+
+            .row-button {
+                font-size: 14px;
+                height: 26px;
+                margin: 4px;
+                width: 26px;
+            }
+        `,
+    ];
+
+    modelNameToViewName(): Map<
+        keyof BasemapSelectorModel,
+        keyof BasemapSelector
+    > {
+        return new Map([
+            ["basemaps", "basemaps"],
+            ["value", "value"],
+        ]);
+    }
+
+    @property({ type: Array }) basemaps: string[] = [];
+    @property({ type: String }) value: string = "";
+    @query('select') selectElement!: HTMLSelectElement;
+
+    render(): TemplateResult {
+        return html`
+            <div class="row-container">
+                <select class="legacy-select" @change=${this.onChange}>
+                    ${this.basemaps.map((basemap) => html`<option>${basemap}</option>`)}
+                </select>
+                <button
+                    class="legacy-button primary row-button close-button"
+                    @click="${this.onCloseClicked}"
+                >
+                    <span class="close-icon material-symbols-outlined">&#xe5cd;</span>
```

Comment:
It's easier to read with the icon specified as icon name instead of code point.

```html
<span class="close-icon material-symbols-outlined">close</span>
```

Reply:
Good call, I changed them all in the toolbar PR, but didn't get around to doing it here.